pyproject.toml

```diff
@@ -20,12 +20,6 @@ where = [ "python" ]
 run_eas = "lsst.ts.eas.eas_csc:run_eas"
 
 [tool.setuptools_scm]
-write_to = "python/lsst/ts/eas/version.py"
```

Comment:
Is this correct? You are still leaving the session here but it will be empty?

Reply:
Yes this is correct. See https://tssw-developer.lsst.io/development-guidelines/language/python.html#python